src/running-a-sora-testnet-node.md

```diff
@@ -0,0 +1,299 @@
+---
+title: "Running a Node on the SORA v3 Testnet | SORA Docs"
```

Comment:
```suggestion
title: "Running a SORA v3 Testnet Node | SORA Docs"
```

Reply:
Resolved in commit 88c29302a00fa18434ec2ba84d3c872148245122